pkgs/by-name/ca/cagebreak/package.nix

```
@@ -23,18 +23,18 @@
   wayland-scanner,
   withXwayland ? true,
   xwayland,
-  wlroots,
+  wlroots_0_18,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "cagebreak";
-  version = "2.3.1";
+  version = "2.4.0";
 
   src = fetchFromGitHub {
     owner = "project-repo";
-    repo = pname;
-    rev = version;
-    hash = "sha256-GAANZIEUtuONPBpk0E3fErgOZtm3wB+gWJNwfO6VOTo=";
+    repo = "cagebreak";
+    rev = finalAttrs.version;
```

```suggestion
    tag = finalAttrs.version;
```
Not blocking, but let's use the opportunity when touching the file already